src/main/scala/nl.knaw.dans.easy.bagstore/package.scala

```diff
@@ -86,4 +86,13 @@ package object bagstore {
       }
     }
   }
+
+  // TODO: canditates for dans-scala-lib?
+  def listDirs(dir: Path): Seq[Path] = {
```

Comment:
Final note about streams here: I'm not sure how it works with resource management and `Stream`. After all, you can't close the resource before you have got all the elements from it...

*Hint: `Observable` would just do that for you...* 😉